_po/ja/how-to/laravel.po

```diff
@@ -411,6 +602,11 @@ msgid ""
 "});\n"
 "*/"
 msgstr ""
+"/* remove\n"
+"Route::get('/', function () {\n"
+"    return view('welcome');\n"
+"});\n"
+"*/"
```

Comment:
Probably, this is needless.
Because this test is same as before translation text.